app/src/main/java/com/example/solom/hotel_csv_app/ItemArrayAdapter.java

```diff
@@ -0,0 +1,61 @@
+package com.example.solom.hotel_csv_app;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ItemArrayAdapter extends ArrayAdapter<String[]>{
+
+    public List<String[]> scoreList = new ArrayList<String[]>();
```

Comment:
Then we can change this to

`public List<Score> mScoreList = new ArrayList<>()`